recipes/jsoncpp/all/patches/implicit-conversion-1.8.3.patch

```diff
@@ -0,0 +1,13 @@
+fixed in 1.9.2 by https://github.com/open-source-parsers/jsoncpp/pull/1023
+
+--- a/src/lib_json/json_value.cpp
++++ b/src/lib_json/json_value.cpp
+@@ -70,7 +70,7 @@ static inline bool InRange(double d, T min, U max) {
+   // The casts can lose precision, but we are looking only for
+   // an approximate range. Might fail on edge cases though. ~cdunn
+   //return d >= static_cast<double>(min) && d <= static_cast<double>(max);
+-  return d >= min && d <= max;
++  return d >= static_cast<double>(min) && d <= static_cast<double>(max);
```

Comment:
Isn't the requirement for this patch contradicting your addition of the `JSONCPP_WITH_WARNING_AS_ERROR` cmake definition?

Reply:
No `JSONCPP_WITH_WARNING_AS_ERROR` is False by default actually, I just want to enforce this value.
But there are still errors as warnings, this why I said that open-source-parsers/jsoncpp#1284 would deserve to be backported in all versions.